### PR TITLE
fix(RELEASE-1068): fix linting issues in create-pyxis-image task

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: tasks/create-internal-request/create-internal-request.yaml [hunk 1]
```diff
@@ -219,7 +219,7 @@ spec:
             jsonpath='{.status.conditions[?(@.type=="Succeeded")].reason}') == "Succeeded" ]
           SH
           chmod +x ${TASKRUN}
-          timeout $(jq -cr '.[]| select(.name=="requestUpdateTimeout") |.value' ${JSON_PARAMS}) ${TASKRUN}
+          timeout "$(jq -cr '.[]| select(.name=="requestUpdateTimeout") |.value' "${JSON_PARAMS}")" "${TASKRUN}"
```
Comment (mmalina) [inline tasks/create-internal-request/create-internal-request.yaml]: This change shouldn't be included here. Also, you have plenty of unrelated commits included in this PR. One onboarding commit and then some merge commits. They should be removed.

**Review by mmalina — COMMENTED** [review]
Other than the extra things in the PR, it looks good.

**Review by johnbieren — APPROVED** [review]

File: tasks/create-pyxis-image/tests/test-create-pyxis-image-multi-containerimages-multi-arch.yaml [hunk 2]
```diff
@@ -90,60 +90,60 @@ spec:
...
               inspect --raw docker://registry.io/multi-arch-image1@sha256:mydigest1
               inspect --raw docker://registry.io/multi-arch-image2@sha256:mydigest2
               inspect --raw docker://registry.io/multi-arch-image3@sha256:mydigest3
               EOF
 
               # check that the actual calls match the expected calls
-              if [ "$(cat $(workspaces.data.path)/skopeo_expected_calls.txt | md5sum)" \
-                != "$(cat $(workspaces.data.path)/mock_skopeo.txt | md5sum)" ]
+              if [ "$(md5sum < "$(workspaces.data.path)/skopeo_expected_calls.txt")" \
+                != "$(md55sum < "$(workspaces.data.path)/mock_skopeo.txt")" ]
```
Comment (johnbieren) [inline tasks/create-pyxis-image/tests/test-create-pyxis-image-multi-containerimages-multi-arch.yaml]: typo - should only be 1 5

**Review by johnbieren — CHANGES_REQUESTED** [review]

**Review by johnbieren — DISMISSED** [review]

**Review by johnbieren — DISMISSED** [review]

File: tasks/create-internal-request/create-internal-request.yaml (hunk 1 above)
Comment (jinqi7) [reply tasks/create-internal-request/create-internal-request.yaml]: Thanks. Updated!

File: tasks/create-pyxis-image/tests/test-create-pyxis-image-one-containerimage-multi-arch.yaml [hunk 3]
```diff
@@ -71,58 +71,58 @@ spec:
...
-              if [ -f $(workspaces.data.path)/mock_cleanup_tags.txt ]; then
+              if [ -f "$(workspaces.data.path)"/mock_cleanup_tags.txt ]; then
                 echo Error: cleanup_tags was not expected to be called. Actual calls:
-                cat $(workspaces.data.path)/mock_cleanup_tags.txt
+                cat "$(workspaces.data.path)/mock_cleanup_tags.txt"
                 exit 1
               fi
 
-              if ! grep -- "--tags testtag" < $(workspaces.data.path)/mock_create_container_image.txt 3> /dev/null
+              if ! grep -- "--tags testtag" < "$(workspaces.data.path)/mock_create_container_image.txt" 3> /dev/null
```
Comment (mmalina) [inline tasks/create-pyxis-image/tests/test-create-pyxis-image-one-containerimage-multi-arch.yaml]: No idea what the `3>` is for. But I googled and apparently you can create your own descriptors with any number higher than 2. But you would need to start them first. 

```
exec 3>output.txt  # Redirect file descriptor 3 to "output.txt"
echo "This will go to the file output.txt" >&3  # Use file descriptor 3
exec 3>&-  # Close file descriptor 3
```

So this looks like a typo. Can you change it to `2>` please? Also in another place.

Comment (jinqi7) [reply tasks/create-pyxis-image/tests/test-create-pyxis-image-one-containerimage-multi-arch.yaml]: Agree. It looks like a typo before and I didn't notice it. Will update. Thanks

**Review by mmalina — DISMISSED** [review]

File: tasks/create-pyxis-image/tests/test-create-pyxis-image-one-containerimage-multi-arch.yaml (hunk 3 above)
Comment (mmalina) [reply tasks/create-pyxis-image/tests/test-create-pyxis-image-one-containerimage-multi-arch.yaml]: Thanks

**Review by johnbieren — APPROVED** [review]
